sources/packages/backend/apps/db-migrations/src/sql/Student/Add-col-disability-status.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE
+    sims.students
+ADD
+    COLUMN IF NOT EXISTS disability_status sims.disability_status;
+
+COMMENT ON COLUMN sims.students.disability_status IS 'Disability status of the student.';
+
+-- Update the disability_status column based on pd_verified and pd_date_sent for existing data.
+UPDATE
+    sims.students
+SET
+    disability_status = CASE
+        WHEN pd_verified IS NULL
+        AND pd_date_sent IS NOT NULL THEN 'Requested'
+        WHEN pd_verified = true THEN 'PD'
```

Comment:
Just wondering, In our today's call, you mentioned that we may be removing `pd_verified` in the future, removing it won't affect this code right? As we did, 
![image](https://github.com/bcgov/SIMS/assets/77353155/967c9473-eead-455d-9b17-861be222413c)
and this is only for  the existing data in the db right?